wormhole-connect/src/views/Bridge/NativeGasSlider.tsx

```diff
@@ -112,6 +115,10 @@ function GasSlider(props: { disabled: boolean }) {
   const { receiving: receivingWallet } = useSelector(
     (state: RootState) => state.wallet,
   );
+  const {
+    usdPrices: { data },
```

Comment:
you could initialize it here 
```suggestion
    usdPrices: { data = {} },
```